src/app_service/service/metrics/service.nim

```diff
@@ -0,0 +1,52 @@
+import json, chronicles, times
+include ../../common/json_utils
+
+import backend/response_type
+import status_go
+import constants
+import ./dto
+
+logScope:
+  topics = "metrics"
+
+proc toggleCentralizedMetrics*(enabled: bool) =
+  try:
+    let payload = %* {"enabled": enabled}
+    let response = status_go.toggleCentralizedMetrics($payload)
+    let jsonObj = response.parseJson
+    if jsonObj.hasKey("error"):
+      error "toggleCentralizedMetrics", errorMsg=jsonObj["error"].getStr
+  except Exception:
+    discard
+
+# for testing, needs to be discussed
+proc addCentralizedMetric*() =
+  try:
+    var metric = CentralizedMetricDto()
+    metric.userId = "123456"
```

Comment:
Any plans on settings this to some real value? 😄 
Probably a ticket to track would be good

UPD: I can see it's "for testing" but anyway

Reply:
The API wil change so, userId will not be needed in next pr.